xbmc/storage/DetectDVDType.cpp

```diff
@@ -420,7 +404,7 @@ int CDetectDVDMedia::DriveReady()
 // Whether a disc is in drive
 bool CDetectDVDMedia::IsDiscInDrive()
```

Comment:
could this method be const?

Reply:
This method doesn't belong here for sure and yes can be const, I plan to rewrite this in the future. Won't be touching this now (see PR description)